docs/source/cli.rst

```diff
@@ -0,0 +1,79 @@
+
+Command-line
```

Comment:
I guess this title is fed into the main doc of the Readthedocs page https://ome-zarr-py--121.org.readthedocs.build/en/121/intro.html ?
Both here, but mainly in https://ome-zarr-py--121.org.readthedocs.build/en/121/intro.html it needs a bit more wording imho, such as 
``Command-Line interface`` or ``Command-line examples`` or similar. Command-line by its own is very terse...

Reply:
Use 'Command line Tools'